src/workbench/SideBarHeader.tsx

```diff
@@ -133,6 +147,42 @@ const SideBarHeader = () => {
             Search
           </Button>
         )}
+        {flags.search && query && (
+          <ButtonGroup
+            isAttached
+            backgroundColor="white"
+            width="full"
+            borderRadius="3xl"
```

Comment:
The focus outline looks very odd. Let's try a pattern similar to https://chakra-ui.com/docs/form/input so the focus ring is around the whole element and the other button is visually inside (via absolute positioning).